src/SFA.DAS.AssessorService.Application/Interfaces/IStandardService.cs

```diff
@@ -12,6 +12,7 @@ public interface IStandardService
 
         Task<StandardCollation> GetStandard(int standardId);
         Task<StandardCollation> GetStandard(string referenceNumber);
+        Task<List<Option>> GetOptions(int stdCode);
```

Comment:
Shouldn't this method be on a IOptionsService/repository interface since it is returning Options rather than Standards.

Or it should return a list of Standards for the stdCode.  Then the caller can extract the Options as required using a linq query.

Reply:
I agree - I attempted to do, but I moved it from the Certificate repository to here so that I could re-used the SQL as that was a definite improvement - attempting to move it to a new separate repository set of a cascade of changes which quickly exceeded the scope of the ticket - there is a separate Tech debt for untangling the mishmash of references coming soon....

Reply:
Refactored the single caller to avoid using this method and removed the method.